### PR TITLE
Add Developer Playground page to docs

### PR DESCRIPTION
**Pull request by igorlukanin**
I've added a new page for *Cube.js Developer Playground*. I would very much like to have such a page to be able to provide links to it in our blog posts and other content.

Also, I moved a couple of *Cube.js CLI* pages to reside under the *Developer Tools* folder together with the newly added *Developer Playground* page. I've also updated a few references and cross-links to *Developer Playground*.

### CODE REVIEW

**Comment by igorlukanin** [comment]
@rpaik Cc'ing you as we discussed that we might add this page.

**Review by rpaik — APPROVED** [review]

File: docs/content/Developer-Tools/Developer-Tools.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+menuOrder: 1
+---
+
+Cube.js comes with a number of developer tools:
+
+* [Cube.js CLI][link-cli]
+* [Cube.js Developer Playground][link-dev-playground]
+
+[link-cli]: /using-the-cubejs-cli
+[link-dev-playground]: /dev-tools/dev-playground
```
Comment (hassankhan) [inline docs/content/Developer-Tools/Developer-Tools.md:14]: Perhaps we could skip this page? It doesn't seem to add much that isn't already linked/referenced from the sidebar.

**Review by hassankhan — APPROVED** [review]
Had one minor comment about the Introduction page, other than that this looks amazing @igorlukanin 🙌

**Comment by hassankhan** [comment]
Posting the Netlify PR link to save everyone a few scrolls: https://deploy-preview-1902--cubejs-docs-staging.netlify.app/

File: docs/content/Developer-Tools/Developer-Tools.md (hunk 1 above)
Comment (igorlukanin) [reply docs/content/Developer-Tools/Developer-Tools.md]: Sure! Added a commit.

**Comment by keydunov** [comment]
Screen Shot 2021-01-26 at 7 31 03 PM

@igorlukanin you are prefixing every page with "Cube.js". I'm not sure It makes a lot of sense. cc @hassankhan

**Comment by igorlukanin** [comment]
@keydunov @hassankhan Are we good right now?

**Comment by keydunov** [comment]
Looks good to me!

**Review by keydunov — APPROVED** [review]

**Pull request merged**